docs/new-command.md

```diff
@@ -5,28 +5,28 @@
 
 This document provides a comprehensive guide for implementing commands in Azure MCP following established patterns.
 
-## Area Pattern: Organizing code by area
+## Toolset Pattern: Organizing code by toolset
```

Comment:
I recommend removing your updates or rebasing as it has changed since this PR was created.